definitions/checks/candlepin/validate_candlepin_db.rb

```diff
@@ -0,0 +1,17 @@
+module Checks::Candlepin
+  class ValidateCandlepinDb < ForemanMaintain::Check
+    metadata do
+      description 'Check to validate candlepin database'
+      tags :pre_upgrade
+      label :validate_candlepin_db
```

Comment:
I'm for the lavel reflecting the naming of the class, so `:candlepin_validate_db` would be better for me: it helps seeing the checks grouped by component, when calling `foreman-maintain health list`